projects/js-packages/config/changelog/fix-vs-code-ts-fixes

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Added "main" entry to package.json to correctly resolve root directory
```

Comment:
This change does not seem to be present anymore.

Reply:
Removed, thanks.